code/obj/item/plants.dm

```diff
@@ -616,6 +618,14 @@ ABSTRACT_TYPE(/obj/item/plant/flower)
 		var/obj/item/reagent_containers/food/snacks/plant/seeds = locate() in src
 		if(seeds)
 			boutput(user, SPAN_NOTICE("You notice some [seeds] fall out of [src]!"))
+			// that's the result if you want to keep 50% of the initial chems in the sunflower at the end.
+			// This accomodates partially for chems injected in the sunflower if some seeds are already missing
+			var/chem_amount = src.reagents.total_volume / (current_seed_count + initial_seed_count)
+			seeds.reagents.maximum_volume = max(chem_amount, seeds.reagents.maximum_volume)
+			seeds.reagents.inert = 1 //let's not explode people by hitting them with a potassium/water sunflower
```

Comment:
But why not? You can already explode them by throwing potassium/water tomatoes at them.